pkg/identity/generic/issuer_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 The Sigstore Authors.
```

Comment:
update the boilerplate here and in other new files to be 

```suggestion
// Copyright 2024 The Sigstore Authors.
```